### PR TITLE
split testbed readme into multiple sections

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: ansible/doc/README.testbed.Internal.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Testbed internals
+
+## Topology definition
+
+ - List of currently defined topologies in veos inventory file
+```
+[servers:vars]
+topologies=[‘t1', ‘t1-lag', 't0', 'ptf32', 'ptf64']
```
Comment (pavel-shirshov) [inline ansible/doc/README.testbed.Internal.md]: It was changed. Latest list is: 't1', 't1-lag', 't0', 'ptf32', 'ptf64', 't0-64', 't0-64-32'

see sonic-mgmt\ansible\veos

File: ansible/doc/README.testbed.Internal.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+ - vm_offset – offset of VM with base configured as vm_base in testbed.csv
+
+ - In this example:
+   - Let’s consider: vm_base == VM0100, vlan_base == ‘100’
+   - First VM:
+     - hostname ARISTA01T1
+     - Uses VM with physical name VM0100
+     - Ethernet1 is connected to vlan 102
+     - Ethernet2 is connected to vlan 103
+     - Ethernet9 is connected to Ethernet9 of ARISTA02T1
```
Comment (pavel-shirshov) [inline ansible/doc/README.testbed.Internal.md]: Need an explanation why? I don't see any '9'-s in the example above. We need a statement that VMs in every topology has a backplane.

File: ansible/doc/README.testbed.Overview.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+- The testbed server has 2 network ports:
+  - Trunk port (Mellanox MT27700 Family CX4) to root fanout switch
+  - Server management port to manage the server, VMs and PTF containers on the server
+ 
+### VMs
+
+The VMs use Arista vEOS. They are using to setup protocol test such as BGP, LACP, LLDP. They are created using ```testbed-cli.sh start-vms``` command. Each VM uses 2G of RAM and has 10 network interfaces:
+
+1. 8 front panel ports. These ports are connected to openvswitch bridges, which are connected to vlan interfaces. The vlan interfaces are connected to the fanout switch (through physical port).
+2. 1 back panel port. All testbed VMs connected to each other using this port (it isn't shown on the figure above).
```
Comment (pavel-shirshov) [inline ansible/doc/README.testbed.Overview.md]: not all VMs are connection through the backplane.
Each topology has its own backplane

File: ansible/doc/README.testbed.Topology.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+- One entry in ```testbed.csv``` consist of:
+  - physical topology: How ports of VMs and PTF connected to DUT
+  - configuration templates for VMs
+- Topologies are defined in vars/topo_*.yml files
+- Current topologies are:
+  - t1: 32 VMs + PTF container for injected ports
+  - t1-lag: 24 VMs + PTF container for injected ports. 8 VMs has two ports each in LAG
+  - ptf32: classic PTF container with 32 ports connected directly to DUT ports
+  - ptf64: as ptf32, but with 64 ports
+  - t0: 4 VMs + ptf. PTF container has 4 injected ports + 28 directly connected ports
```
Comment (pavel-shirshov) [inline ansible/doc/README.testbed.Topology.md]: not all topologies are described here. t0-64 and t0-64-32 have to be describer

File: ansible/doc/README.testbed.Internal.md (hunk 1 above)
Comment (lguohan) [inline ansible/doc/README.testbed.Internal.md]: the main change is to split the document into several sections. will address new topology later.

File: ansible/doc/README.testbed.Topology.md (hunk 4 above)
Comment (lguohan) [inline ansible/doc/README.testbed.Topology.md]: will ad new topology in separate cr.

File: ansible/doc/README.testbed.Internal.md (hunk 2 above)
Comment (lguohan) [inline ansible/doc/README.testbed.Internal.md]: added statement in the overview.

File: ansible/doc/README.testbed.Overview.md (hunk 3 above)
Comment (lguohan) [inline ansible/doc/README.testbed.Overview.md]: change.d

**Pull request merged**